tests/ci/run_benchmark_build_tests.sh

```diff
@@ -23,25 +23,27 @@ function build_aws_lc_fips {
       -DFIPS=1 \
       -DENABLE_DILITHIUM=ON \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+      -DBUILD_SHARED_LIBS=1 \
       -DBUILD_TESTING=OFF
   pushd "$BUILD_ROOT"
   ninja install
   popd
 }
 
-function build_aws_lc_fips_2022 {
-    echo "building the fips-2022-11-02 branch of aws-lc in FIPS mode"
-    git clone --depth 1 --branch fips-2022-11-02 https://github.com/aws/aws-lc.git "${scratch_folder}/aws-lc-fips-2022-11-02"
-    pushd "${scratch_folder}/aws-lc-fips-2022-11-02"
+function build_aws_lc_branch {
+    branch="$1"
```

Comment:
nit: should we default to something or error out if no branch is provided to this function?